team.html

```diff
@@ -254,6 +254,34 @@ <h6 class="card-subtitle pt-1 pb-3" style="color: #EF4035">Treasurer</h6>
                         </div>
                      </div>
                   </div>
+
+                  <div class="card my-2">
+                     <img class="card-img-top p-2" src="" alt="Manisha Kusuma">
```

Comment:
Add your own image (make sure you change the "alt" text). Also, make sure that the image is compressed.